src/JuMP.jl

```diff
@@ -409,6 +414,27 @@ function _try_get_solver_name(model_like)
     end
 end
 
+"""
+    set_string_names(model::Model, value::Bool)
```

Comment:
I'd recommend `set_string_names_on_creation` for more verbosity. This will appear in advanced code and only in few spots, so it doesn't hurt to be clearer. Same for the name of the field of `Model`.

Reply:
Done